### PR TITLE
Bump d2l-activities version

### PR DESCRIPTION
**Pull request by jeroach**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2608,7 +2608,7 @@
       }
     },
     "d2l-activity-alignments": {
-      "version": "github:Brightspace/d2l-activity-alignments#5f92da97a311194de01b3aed01e473122f46072a",
+      "version": "github:Brightspace/d2l-activity-alignments#2c2583f20d8662e44b93fa6c1523658e97a554da",
```
Comment (jeroach) [inline package-lock.json]: @djwester fyi

File: package-lock.json [hunk 2]
```diff
@@ -2645,7 +2645,7 @@
       }
     },
     "d2l-button": {
-      "version": "github:BrightspaceUI/button#9552381a355c576b32222e80db577a1ce746b8b2",
+      "version": "github:BrightspaceUI/button#d44b842cc16e307b1782b3bfed8e0e04910e1972",
```
Comment (jeroach) [inline package-lock.json]: @mdgbayly fyi

**Comment by mtjandra** [comment]
Quick skim look exactly the same as https://github.com/Brightspace/brightspace-integration/pull/1326

That one has your activities change in it too

File: package-lock.json [hunk 3]
```diff
@@ -3124,11 +3124,12 @@
       }
     },
     "d2l-outcomes-level-of-achievements": {
-      "version": "github:Brightspace/outcomes-level-of-achievement-ui#dce9e572dea085743d28f2b169835f5066407c01",
+      "version": "github:Brightspace/outcomes-level-of-achievement-ui#ef2b4b3064e7a5b60d6a28804bb4fe32ad10a8fb",
```
Comment (jeroach) [inline package-lock.json]: @rgoessl fyi

File: package-lock.json [hunk 4]
```diff
@@ -3175,7 +3176,7 @@
       }
     },
     "d2l-rubric": {
-      "version": "github:Brightspace/d2l-rubric#7c300ac93ca4e3eeb85b5888c2837edc40a7bad3",
+      "version": "github:Brightspace/d2l-rubric#6d8ef12ce6d76d0450ac64391f0aa11c4f84567d",
```
Comment (jeroach) [inline package-lock.json]: @djwester fyi

File: package-lock.json (hunk 1 above)
Comment (djwester) [reply package-lock.json]: I think there is another PR Up right now that has this change in it too.  Might want to talk to @rgoessl to avoid conflicts

**Pull request merged**